### DISCUSSION TITLE
:CocList Command Failed

### DISCUSSION THREAD

**Discussion by danniserena** [neoclide/coc.nvim#5027] [Q&A]
When I use `:CocList` like `:CocList extensions` or `:CocList services`, a piece of error message pop up, here is the log info:

![image](https://github.com/neoclide/coc.nvim/assets/73869793/0cf6fa24-d111-4da3-8919-42c0f17ffdd4)

coc.vim used to work fine, so I guess sth I did change my nvim or my shell ..., but I don't know how to fix it
I tried to re-download coc.nvim using **Packer**, and even tried to manually install it, but it didn't work
here is my settings:
```
vim version: NVIM v0.10.0
node version: v16.20.2
coc.nvim version: 0.0.82-ba3c0a94 2024-05-30 20:05:24 +0800
coc.nvim directory: /root/.local/share/nvim/site/pack/coc/start/coc.nvim
term: xterm
platform: linux
```

**Comment by fannheyward** [comment]
Thank you for your feedback, I've tested on macOS with nvim 0.10, node 16.20.2, with same coc.nvim version as you're using (the master branch), can't reproduce this.

Can you test coc.nvim's `release` branch?
